app/scraper.py

```diff
@@ -1,171 +0,0 @@
-from __future__ import print_function
```

Comment:
Looks like you created one file for each scraper but you have not added other scrapers here.

Reply:
This has already been merged. This file has been removed, and the initialization has been added to `__init__.py` (check  #141)